apps/meteor/client/views/room/MessageList/MessageList.tsx

```diff
@@ -72,7 +73,8 @@ export const MessageList = ({ rid }: MessageListProps): ReactElement => {
 										data-unread={firstUnread}
 										data-sequential={sequential}
 										sequential={shouldShowAsSequential}
-										message={message}
+										message={message as IThreadMessage}
+										_subscription={userSubscription}
```

Comment:
Any reason to add the underscore to this prop?

Reply:
no, I'll change it here